CHANGELOG.md

```diff
@@ -14,6 +14,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - Under unlikely state restoration circumstances, a user who cleared their data could unexpectedly experience the data clear multiple times (#39)
 
+## [1.3.1] - 2019-07-08
```

Comment:
nit: this date should be the date released to consumer, not the day we built the build so we can e.g. identify trends in data from releasing the new build. Ideally, Amazon will tell us when they release and we can use that date. For now I'd just write 2019-07-?